app/experimenter/experiments/tests/test_migrations.py

```diff
@@ -1,47 +1,104 @@
+        VariantPreferences = self.old_state.apps.get_model(
+            "experiments", "VariantPreferences"
+        )
+        RolloutPreference = self.old_state.apps.get_model(
+            "experiments", "RolloutPreference"
         )
-        NimbusBranchFeatureValue.objects.create(branch=branch_with_value, value="value")
 
-        branch_without_value = NimbusBranch.objects.create(
-            experiment=experiment,
-            slug="treatment",
+        user = User.objects.create(email="test@example.com")
+        experiment = Experiment.objects.create(slug="test_experiment")
+        variant = ExperimentVariant.objects.create(
+            experiment_id=experiment.id, slug="test_variant"
+        )
+        ExperimentEmail.objects.create(experiment_id=experiment.id, type="test_email")
+        ExperimentComment.objects.create(
+            experiment_id=experiment.id, created_by_id=user.id, text="test_comment"
+        )
+        ExperimentChangeLog.objects.create(
+            experiment_id=experiment.id, changed_by_id=user.id, new_status="test_status"
+        )
+        VariantPreferences.objects.create(
+            variant_id=variant.id,
+            pref_branch="test_branch",
+            pref_name="test_name",
+            pref_type="test_type",
+            pref_value="test_value",
+        )
+        RolloutPreference.objects.create(
+            experiment_id=experiment.id,
+            pref_name="test_name",
+            pref_type="test_type",
+            pref_value="test_value",
         )
-        NimbusBranchFeatureValue.objects.create(branch=branch_without_value, value=None)
 
     def test_migration(self):
         """Run the test itself."""
-        NimbusBranchFeatureValue = self.new_state.apps.get_model(
-            "experiments", "NimbusBranchFeatureValue"
+        User = self.new_state.apps.get_model("auth", "User")
+        Experiment = self.new_state.apps.get_model("legacy_experiments", "Experiment")
+        ExperimentVariant = self.new_state.apps.get_model(
+            "legacy_experiments", "ExperimentVariant"
+        )
+        ExperimentEmail = self.new_state.apps.get_model(
+            "legacy_experiments", "ExperimentEmail"
+        )
+        ExperimentComment = self.new_state.apps.get_model(
+            "legacy_experiments", "ExperimentComment"
+        )
+        ExperimentChangeLog = self.new_state.apps.get_model(
+            "legacy_experiments", "ExperimentChangeLog"
+        )
+        VariantPreferences = self.new_state.apps.get_model(
+            "legacy_experiments", "VariantPreferences"
+        )
+        RolloutPreference = self.new_state.apps.get_model(
+            "legacy_experiments", "RolloutPreference"
         )
 
-        self.assertFalse(NimbusBranchFeatureValue.objects.filter(value=None).exists())
-        self.assertTrue(NimbusBranchFeatureValue.objects.filter(value="value").exists())
-        self.assertTrue(NimbusBranchFeatureValue.objects.filter(value="").exists())
```

Comment:
Are we no longer `assert`ing in these tests? Does `XXX.objects.get` cover these cases for us?